### DISCUSSION TITLE
404 not found Error 54113

### DISCUSSION THREAD

**Discussion by samartinell** [community/community#112288] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

So everything worked great yesterday, one of my programs relies on a config file that I download each time from releases in my project.
Today it stopped working for no reason, at least nothing I could think of. Doesn't seem like Github is down or something, and I can't find anything online about this error.
![image](https://github.com/community/community/assets/48961365/a71da405-8c38-4db0-b9b0-6dbaa72fcca6)
Here's how it looks when I click on a download link, it redirects me to some strange URL, you can see a part of it on a screenshot.
It is not a browser issue how it might look by the error (Varnish cache server), I tried in a different browser, on a different device, different network. Nothing worked and I don't know why, anyone had similar issue?

**Comment by LadyDanika** [comment]
I'm also seeing this error.

**Comment by samartinell** [comment, marked as answer]
Well, it fixed itself over time. and I already did a little backup in my program so there's less chance of a complete failure again. I'm guessing one of the servers that stored my file broke or something and they fixed it in a day, so if anyone has the same problem, I think there's nothing you can do but wait.

**Comment by IbukiMoriya** [comment]
It's fixed at my place too!
